json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+include_once JETPACK__PLUGIN_DIR . 'modules/protect/shared-functions.php';
+
```

Comment:
@lezama is it okay to include on this endpoint the shared functionality from the Protect module? Or will that break on wordpress.com?